gh-pages/Gemfile.lock

```diff
@@ -8,10 +8,10 @@ GEM
       tzinfo (~> 1.1)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    algolia_html_extractor (2.6.2)
+    algolia_html_extractor (2.6.4)
       json (~> 2.0)
-      nokogiri (~> 1.10.4)
-    algoliasearch (1.27.1)
+      nokogiri (~> 1.10)
```

Comment:
Won't this continue to complain as well?

Reply:
No, it doesn't: "~> 1.10" means >= 1.10 and < 2.0.